commit d99c74e2913ab098953bd5b422b95f39e2dcfb55
Author: Sam Vilain <sam.vilain@catalyst.net.nz>
Date:   Fri Sep 21 14:02:34 2007 +1200

    git-svn: fix test for trunk svn (transaction out of date)
    
    Older svn clients did not raise a 'transaction out of date' error here, but
    trunk does - so 'svn up'.
    
    Signed-off-by: Sam Vilain <sam.vilain@catalyst.net.nz>
    Acked-by: Eric Wong <normalperson@yhbt.net>
    Signed-off-by: Junio C Hamano <gitster@pobox.com>

diff --git a/t/t9101-git-svn-props.sh b/t/t9101-git-svn-props.sh
index 622ea1c..5aac644 100755
--- a/t/t9101-git-svn-props.sh
+++ b/t/t9101-git-svn-props.sh
@@ -140,6 +140,7 @@ test_expect_success 'test show-ignore' "
 	cd test_wc &&
 	mkdir -p deeply/nested/directory &&
 	svn add deeply &&
+	svn up &&
 	svn propset -R svn:ignore 'no-such-file*' .
 	svn commit -m 'propset svn:ignore'
 	cd .. &&
